### PR TITLE
Add scripts to run examples/kubernetes/*

### PR DESCRIPTION
**Pull request by VamshiTeja**
## Motivation
Write a script to run examples/kubernetes
Related #1493 

## Description of the changes
- [x] Write scripts to run examples/kubernetes/mlflow and examples/kubernetes/simple
- [x] Update Readme accordingly

### CODE REVIEW

File: examples/kubernetes/mlflow/README.md [hunk 1]
```diff
@@ -10,33 +10,27 @@ This example's code is based on ../../pytorch_lightning_simple.py example with t
 
 In order to run this example you have to do the following steps:
 
-1. (Optional) If run locally inside [minikube](https://github.com/kubernetes/minikube) you have to use the Docker daemon inside of it:
+First run `run.sh` which takes two arguments $IsMinikube and $IMAGE_NAME
```
Comment (crcrpar) [inline examples/kubernetes/mlflow/README.md:13]:
```suggestion
First run `run.sh` which takes two arguments `$IsMinikube` and `$IMAGE_NAME`
```

File: examples/kubernetes/mlflow/README.md [hunk 2]
```diff
@@ -10,33 +10,27 @@ This example's code is based on ../../pytorch_lightning_simple.py example with t
...
 
-3. Apply the kubernetes manifests:
+ ```bash
+$ sh run.sh True optuna-kubernetes-mlflow:example
+ ```
 
-```bash
-$ kubectl apply -f k8s-manifest.yaml
-```
+- If you want to run in cloud, please change the IMAGE_NAME accordingly in k8s-manifest.yaml and run as follows. Also please make sure that you kubernetes context is set correctly.
```
Comment (crcrpar) [inline examples/kubernetes/mlflow/README.md:21]:
```suggestion
- If you want to run in cloud, please change the `IMAGE_NAME` accordingly in k8s-manifest.yaml and run as follows. Also please make sure that your kubernetes context is set correctly.
```

File: examples/kubernetes/mlflow/run.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  fi
+
+  # Deploy to cluster
+  echo "\nDeploying example to cluster \n"
+  kubectl apply -f k8s-manifest.yaml
+}
+
+IsMinikube=$1
+IMAGE_NAME=$2
+run ${IsMinikube} ${IMAGE_NAME}
```
Comment (crcrpar) [inline examples/kubernetes/mlflow/run.sh:27]: Could you end this file with a newline?

File: examples/kubernetes/simple/README.md [hunk 4]
```diff
@@ -10,26 +10,21 @@ in order to avoid errors when the code is run from multiple workers.
 
 In order to run this example you have to do the following steps:
 
-1 - (Optional) If run locally inside [minikube](https://github.com/kubernetes/minikube) 
-you have to use the Docker daemon inside of it: 
+Run `run.sh` which takes two arguments $IsMinikube and $IMAGE_NAME
```
Comment (crcrpar) [inline examples/kubernetes/simple/README.md:13]:
```suggestion
Run `run.sh` which takes two arguments `$IsMinikube` and `$IMAGE_NAME`
```

File: examples/kubernetes/simple/run.sh [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  fi
+
+  # Deploy to cluster
+  echo "\nDeploying example to cluster \n"
+  kubectl apply -f k8s-manifests.yaml
+}
+
+IsMinikube=$1
+IMAGE_NAME=$2
+run ${IsMinikube} ${IMAGE_NAME}
```
Comment (crcrpar) [inline examples/kubernetes/simple/run.sh:27]: Same as `mlflow/run.sh`, could you end this file with a newline?

**Comment by VamshiTeja** [comment]
> Thank you for your swift work!
> 
> I confirmed both `run.sh` work correctly in my local environment, i.e., minikube.
> Out of curiosity, did you run these examples on some public cloud service as the docs explain how to run in cloud?

Thank You for your quick review! I didn't get a chance to run them in cloud but would be happy to help if any issue comes up in this regard.

**Pull request merged**